service/controller/azure/v14/version_bundle.go

```diff
@@ -8,9 +8,9 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
-				Kind:        versionbundle.KindChanged,
+				Component:   "cluster-operator",
```

Comment:
@tomahawk28 The message needs to be updated for Azure and KVM. Please do a follow up PR.